scripts/run_tests.sh

```diff
@@ -8,11 +8,14 @@ set -e
 echo "Starting Validator"
 ./scripts/run_validator.sh
 
-echo "Running schema tests"
-./scripts/test_schemas.sh schemas/en
+echo "Running schema validation"
+./scripts/validate_test_schemas.sh test_schemas/en
```

Comment:
`test_language.json` is in `test_schemas/cy` so we can remove passing the dir to test all `test_schemas`.

Reply:
We can't as validator will complain about special characters in the welsh schemas.

Reply:
Ah ya but in that case, this needs to change to `./test_schemas/en` in here:
https://github.com/ONSdigital/eq-questionnaire-runner/blob/use-github-actions/scripts/validate_test_schemas.sh#L31